index.js

```diff
@@ -0,0 +1,50 @@
+const submitButton = document.getElementById("submit-btn");
+const displayResult = document.getElementById("display-result");
+const locationInput = document.getElementById("location-input");
+
+const key = "c7e92242bb16d357d12aab3eb76e6d3e";
+
+submitButton.addEventListener("click", handleSearch);
+function handleSearch() {
+  renderLoading();
+  getData(locationInput.value);
+}
+locationInput.addEventListener("keypress", function (event) {
+  // If the user presses the "Enter" key on the keyboard
+  if (event.key === "Enter") {
+    event.preventDefault();
+    submitButton.click();
+  }
+});
+
+async function getData(cityName) {
+  let textToRender = "";
+  await fetch(
+    `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=${key}`
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      if (data.main) {
+        let tempCelc = (data.main.temp - 273.15).toFixed(2);
```

Comment:
since you are not reassigning any value to `tempCelc`, declare it using `const` instead of `let`